src/Extension/Settings/ExtensionOptions.cs

```diff
@@ -38,12 +38,12 @@ public class ExtensionOptionsPage : BaseOptionPage<CodigaOptions>
 	public class CodigaOptions : BaseOptionModel<CodigaOptions>
 	{
 		[Category("Codiga")]
-		[DefaultValue(true)]
-		public bool UseCodingAssistant { get; set; } = true;
+		[DefaultValue(false)]
+		public bool UseCodingAssistant { get; set; } = false;
```

Comment:
This is for the shortcuts correct? It does not impact anything else?

Reply:
As far as I've seen, it doesn't affect anything else.